scout/parse/variant/variant.py

```diff
@@ -458,10 +458,10 @@ def set_found_db(found_db_info: Optional[str]) -> Optional[List[str]]:
         str, replace_nan(variant.INFO.get("TRANSCRIPT_ID_B", ""))
     )
     parsed_variant["exon_number_a"] = call_safe(
-        str, call_safe(int, replace_nan(variant.INFO.get("EXON_NUMBER_A", "")))
+        int, call_safe(int, replace_nan(variant.INFO.get("EXON_NUMBER_A", "")))
```

Comment:
these are numbers but they get store into the exon field of transcripts which is a string

Reply:
But in the scout document should be numbers again, no?

Reply:
Checked other variant types, like SVs and exons are strings in there (i.e. "2/7"). So I've reverted these values as they were before